instruments/thorlabs/pm100usb.py

```diff
@@ -93,7 +93,7 @@ def __init__(self, parent):
             # We want flags to be a named tuple over bools.
             # pylint: disable=protected-access
             self._flags = parent._SensorFlags(**{
-                e.name: bool(e & self._flags)
+                e.name: bool(e & int(self._flags))
```

Comment:
From right above, `self._flags` is still formatted as a string, while `e` is the value of `SensorFlag`, which in itself is an `IntEnum`. Thus `self._flags` must be compared to an integer here.